### PR TITLE
🎨 Improvements top bar and notifications 

### PR DESCRIPTION
**Pull request by AmandaElvkull**
Improved design fixes top bar: 
- update to correct gap between topbar actions
- change from side bar to topbar menu for noticiations
- close top bar menus on resize window. 
-  small design fixes

### CODE REVIEW

File: src/components/Navigation/TopBar/TopBar.styles.tsx [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import { Button } from '@equinor/eds-core-react';
 import { tokens } from '@equinor/eds-tokens';
 
+import { UnreadRedDot } from 'src/components/Navigation/TopBar/Notifications/Notifications';
```
Comment (aslakihle) [inline src/components/Navigation/TopBar/TopBar.styles.tsx:4]: This import could be shortened. 
 './Notifications/Notifications' i think (not 100% sure so did not do suggestion 🙈😂 )

**Review by aslakihle — CHANGES_REQUESTED** [review]

File: src/components/Navigation/TopBar/Notifications/Notifications.test.tsx [hunk 2]
```diff
@@ -159,11 +159,12 @@ test('renders button and panel correctly', async () => {
   const icons = screen.getAllByTestId('eds-icon-path');
 
   expect(icons[0]).toHaveAttribute('d', notifications.svgPathData);
-  expect(screen.getByTestId('side-panel')).toHaveStyleRule('display', 'none');
+
+  expect(screen.queryByTestId('top-bar-menu')).not.toBeInTheDocument();
```
Comment (aslakihle) [inline src/components/Navigation/TopBar/Notifications/Notifications.test.tsx:164]: Hei

File: src/components/Navigation/TopBar/Notifications/Notifications.tsx [hunk 3]
```diff
@@ -66,6 +35,8 @@ export const UnreadRedDot = styled.div`
   right: 2px;
   top: 4px;
   border: 2px solid ${colors.text.static_icons__primary_white.rgba};
+  // Box-sizing is a quickfix for use in PWEX because of global styling
```
Comment (mariush2) [inline src/components/Navigation/TopBar/Notifications/Notifications.tsx:38]: Good idea to have a comment here to explain 🥳

**Review by mariush2 — CHANGES_REQUESTED** [review]
Smaller stuff

**Review by mariush2 — APPROVED** [review]

**Review by aslakihle — APPROVED** [review]

**Pull request merged**